server/modules/modules.go

```diff
@@ -147,7 +183,52 @@ func Auth(message *messageService.Message, conn *websocket.Conn) {
 		UserName: us.UserName,
 		Token:    tok,
 	}
-	sendMsg(conn, newMessageHeader)
+	nick, err := auth.SendNickName(user)
+	if err != nil {
+		loger.Log.Warningf("Can`t get user nick_name. %s", err)
+		return
+	}
+	newMessageBody := messageService.Authentification{
+		NickName: nick,
+	}
+	newRawMessageBody, err := json.Marshal(newMessageBody)
+	sendMsg(conn, newMessageHeader, newRawMessageBody)
+}
+
+func SendContacts(message *messageService.Message, conn *websocket.Conn) {
+	// For testing
```

Comment:
I need this for testing if Volodymyr change something on his side to check work of this module